tools/validate_configs/os_compatibility_tests/batch-lustre.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: test-batch-lustre
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: test
+  region: us-central1
+  zone: us-central1-a
+  # image:
+  #   family: hpc-centos-7
+  #   project: centos-cloud
+  image:
+    family: hpc-rocky-linux-8
+    project: rocky-linux-cloud
+
+deployment_groups:
+- group: primary
+  modules:
+
+  ###########
+  # Network #
+  ###########
+
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  # Example - ./resources/network/vpc
+  - id: network1
+    source: modules/network/vpc
+
+  ###########
+  # Storage #
+  ###########
+
+  # This file system has an associated license cost.
+  # https://console.developers.google.com/marketplace/product/ddnstorage/exascaler-cloud
+  - id: lustre
+    source: community/modules/file-system/DDN-EXAScaler
+    use:
+    - network1
+    settings:
+      local_mount: /lustre
+      mgs:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 1
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+      mds:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 1
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+      oss:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 3
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+
+  ###############
+  # Cloud Batch #
+  ###############
+
+  - id: batch-job
+    source: modules/scheduler/batch-job-template
+    use:
+    - network1
+    - lustre
+    settings:
+      runnable: "sudo echo 'hello, world!' > /lustre/test-from-batch.txt"
+      machine_type: n2-standard-4
+
+  - id: batch-login
+    source: modules/scheduler/batch-login-node
+    use:
+    - batch-job
+    - lustre
```

Comment:
Do not use lustre here. It will automatically mount any fs used by the job. I think this is why you were having the error with luster and batch.